### PR TITLE
OCPBUGS-3516: fix CSV unmarshaling error with custom unmarshaler with pretty errors (#1138)

### PR DESCRIPTION
**Pull request by grokspawn**
Also including a change to root .gitignore for test dockerfiles created during repo verification steps.

### CODE REVIEW

**Comment by grokspawn** [comment]
/retest
Pod readiness was not something that this PR should have impacted.

File: staging/operator-registry/alpha/declcfg/declcfg.go [hunk 1]
```diff
@@ -6,6 +6,8 @@ import (
 	"errors"
 	"fmt"
 
+	prettyunmarshaler "github.com/operator-framework/operator-registry/pkg/prettyunmarshaler"
```
Comment (stevekuznetsov) [inline staging/operator-registry/alpha/declcfg/declcfg.go:9]: nit: these import aliases are not needed

**Review by stevekuznetsov — COMMENTED** [review]
Shouldn't this be a cherry-pick of an upstream commit?

**Comment by stevekuznetsov** [comment]
Looks like it is a cherrypick. Let's get it in, in order, so future upstream syncs to master branch are not out-of-order.

/hold

**Comment by stevekuznetsov** [comment]
@grokspawn It is likely not worth your time to re-test this, as it should be pulled in via a sync, not on it's own.

**Comment by stevekuznetsov** [comment]
Now part of #562

**Pull request closed without merging**